### PR TITLE
Raises the galaxy-importer version requirement

### PR DESCRIPTION
**Pull request by dkliban**
re: #9346

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 async_lru~=1.0
-galaxy_importer>=0.3.1
+galaxy_importer>=0.4.1
```
Comment (fao89) [inline requirements.txt:2]: 👍🏽

**Review by fao89 — APPROVED** [review]

**Review by mdellweg — CHANGES_REQUESTED** [review]
Can you please also remove installing of galaxy-importer from the before-install script? I believe we want to treat that as an external dependency, and so we should only test against released versions. I think, this is what broke the oci nightly image for several days.

**Review by mdellweg — APPROVED** [review]
I see we are installing it via the plugin-template now, which will install from pypi in a release workflow. That will prevent broken releases of this type at least.

**Comment by fao89** [comment]
merging to see if it fixes pulp_installer CI:
https://github.com/pulp/pulp_installer/actions/runs/1451438364

**Pull request merged**